src/net.h

```diff
@@ -417,6 +410,26 @@ class CNode
     std::atomic_bool fPauseRecv{false};
     std::atomic_bool fPauseSend{false};
 
+    const ConnectionType& GetConnectionType() const
+    {
+        return m_conn_type;
+    }
+
+    /** Move all messages from the received queue to the processing queue. */
+    void MarkReceivedMsgsForProcessing(unsigned int recv_flood_size)
+        EXCLUSIVE_LOCKS_REQUIRED(!m_msg_process_queue_mutex);
+
+    /** Poll the next message from the processing queue of this connection. */
+    std::optional<std::pair<CNetMessage, bool>> PollMessage(size_t recv_flood_size)
+        EXCLUSIVE_LOCKS_REQUIRED(!m_msg_process_queue_mutex);
+
+    /** Account for the total size of a send message in the per msg type connection stats. */
+    void AccountForSendBytes(const std::string& msg_type, size_t received_bytes)
```

Comment:
`s/Send/Sent/`
`s/received_bytes/sent_bytes/`

```suggestion
    void AccountForSentBytes(const std::string& msg_type, size_t sent_bytes)
```